### PR TITLE
Make named avatars persistent

### PR DESCRIPTION
**Pull request by HoeenCoder**

### CODE REVIEW

File: js/client-chat.js [hunk 1]
```diff
@@ -881,13 +881,8 @@
...
 			case 'avatar':
 				var parts = target.split(',');
-				var avatarString = toId(parts[0]);
-				var avatar = parseInt(avatarString, 10);
-				if (avatar) {
-					Dex.prefs('avatar', avatar);
-				} else if (['bw2elesa', 'teamrocket', 'yellow', 'zinnia', 'clemont'].indexOf(avatarString) > -1) { // custom avatar exceptions
-					Dex.prefs('avatar', '#' + avatarString);
-				}
+				var avatar = toId(parts[0]);
```
Comment (Zarel) [inline js/client-chat.js]: Close, but avatars can have `-` in them.

Comment (Zarel) [reply js/client-chat.js]: The server is currently using a hardcoded regex, you can just copy/paste it.

Comment (HoeenCoder) [reply js/client-chat.js]: Whoops, slipped my mind when doing this.

File: js/client-chat.js [hunk 2]
```diff
@@ -881,13 +881,8 @@
 
 			case 'avatar':
 				var parts = target.split(',');
-				var avatarString = toId(parts[0]);
-				var avatar = parseInt(avatarString, 10);
-				if (avatar) {
-					Dex.prefs('avatar', avatar);
-				} else if (['bw2elesa', 'teamrocket', 'yellow', 'zinnia', 'clemont'].indexOf(avatarString) > -1) { // custom avatar exceptions
-					Dex.prefs('avatar', '#' + avatarString);
```
Comment (Slayer95) [inline js/client-chat.js]: What about #avatars?

Comment (HoeenCoder) [reply js/client-chat.js]: No longer "custom" (#), they are now where all the other avatars are. eg: https://play.pokemonshowdown.com/sprites/trainers-byname/yellow.png

True custom avatars (#hoeenhero) aren't set through this command anyways.

Comment (Zarel) [reply js/client-chat.js]: They probably will be, in the future. But it's not really a problem for now.

**Pull request merged**